samples/JwtSample/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:57780/",
+      "applicationUrl": "http://localhost:8719/",
```

Comment:
I don't know what I did to make VS do this...

![image](https://user-images.githubusercontent.com/7574/36516096-55fd3b94-1731-11e8-97d3-ef46e15ce77b.png)